docs/pages/desktop-access/reference/sessions.mdx

```diff
@@ -87,4 +76,10 @@ player supports toggling between play and pause, but does not support seeking to
 a specific point in the stream, rewinding, or restarting playback when the end
 of the stream is reached. To replay a session, refresh the page.
 
+## Log rotation
 
+Be aware, Desktop Access session recording comprises of raw PNG-based files,
+and are stored on the host running the Desktop Access service. Session recordings
```

Comment:
As mentioned offline, they are not stored on the desktop service long term, and in some cases are not stored on that host at all (sync recording)

I would rephrase this paragraph. Even the title of "Log rotation" is a little confusing because there are so many things in teleport that can be considered a log.

Let's say something like:

> Be aware, Desktop Access session recordings capture the entire contents of the
> screen. This means they take up significantly more disk space than SSH or Kubernetes
> session recordings. When using async recording modes, ensure that the host running
> Teleport's desktop service has sufficient disk space to store recordings that are
> in progress. If disk space is a concern, we recommend using sync recording and
> configuring a cloud storage solution such as S3.

Reply:
I took your paragraph, but added after the line linking to the config reference.